src/DependencyInjection/Configuration.php

```diff
@@ -437,6 +440,50 @@ private function createClientPluginNode()
                         ->end()
                     ->end()
                 ->end()
+                ->arrayNode('vcr')
+                    ->canBeEnabled()
+                    ->addDefaultsIfNotSet()
+                    ->info('Record response to be replayed during tests or development cycle.')
+                    ->validate()
+                        ->ifTrue(function ($config) {
+                            return 'filesystem' === $config['recorder'] && empty($config['fixtures_directory']);
+                        })
+                        ->thenInvalid('If you want to use the "filesystem" recorder you must also specify a "fixtures_directory".')
+                    ->end()
+                    ->children()
+                        ->enumNode('mode')
+                        ->info('What should be the behavior of the plugin?')
+                        ->values(['record', 'replay', 'replay_or_record'])
+                        ->defaultValue('replay_or_record')
```

Comment:
the defensive value would be to default to "replay". i am inclined to make this have no default and be a required field to force people to actively decice which mode is right for them (if they enable the vcr at all)

Reply:
I removed the default value